### DISCUSSION TITLE
Help with Spotify Playlists

### DISCUSSION THREAD

**Discussion by vitospinelli** [MiczFlor/RPi-Jukebox-RFID#1770] [Q&A]
Hello,
First of all thank you so much for this wonderful project.

I'm trying to build a phoniebox for my son. I installed everything and all seems to work (Rp zero 2, sound card, Rfid, buttons etc...).

I can add local files and also spotify tracks. Swiping the rfid cards starts the playback as expected. However when I try to add a Spotify playlist or album, it doesn't work. I register a new card, add it to a folder, paste the playlist URI. However when I swipe the card or simply click play nothing happens. 

I searched all over the place but it seems nobody else experienced this issue. What could it be?? Please help!
Vito

**Comment by vitospinelli** [comment]
I just tried a fresh reinstall but I had no success...

**Reply by vitospinelli** [reply]
Ok I think I found the issue. Even though I can load single tracks and in Iris it shows that I'm correctly logged into spotify, the command journalctl -u mopidy shows an authorization error. It asks me to get new credentials and says "invalid_client Client not known". Any idea how I can solve this?

**Reply by vitospinelli** [reply]
This is starting to sound like a monologue, but I found the solution. I was using the client Id and client secret of the Spotify app. Instead I should have authorized mopidy and inserted the client Id and client secret provided by Mopidy. Everything seems to be working now. I leave this here in case it may be useful for someone else. This part was not very clear for me in the installation process. Thanks for the great work! I hope this project stays strong.

**Comment by MiczFlor** [comment]
Hi @vitospinelli 
there is nothing like a good monologue. Usually that means: you did it :)
And if you did it, possibly a good place for your findings would be the wiki?
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki
?
All the best, micz
